app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/MainActivity.java

```diff
@@ -150,7 +150,7 @@ protected void onCreate(Bundle savedInstanceState) {
             SharedPreferences.Editor editor = sharedPref.edit();
             editor.putInt(getString(R.string.mainFileVersion), 0);
             editor.putInt(getString(R.string.patchFileVersion), 0);
-            editor.commit();
+            editor.apply();
```

Comment:
This change needs to be reverted since editor.commit() is necessary

Reply:
Done.